src/main/components/form/claimDetailsValidator.ts

```diff
@@ -0,0 +1,70 @@
+export const validatePersonalDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+
+  const {
+    typeOfClaim,
+    reasonableAdjustments,
+    personalDetailsCheck,
+    hearingPreferences,
+    claimantContactPreference,
+    claimantContactLanguagePreference,
+    claimantHearingLanguagePreference,
+    address1,
+    addressTown,
+    addressPostcode,
+  } = userCase;
+
+  if (
+    !typeOfClaim ||
+    !reasonableAdjustments ||
+    !personalDetailsCheck ||
+    !hearingPreferences ||
+    !claimantContactPreference ||
+    !claimantContactLanguagePreference ||
+    !claimantHearingLanguagePreference ||
+    !address1 ||
+    !addressTown ||
+    !addressPostcode
+  ) {
+    return false;
+  }
+
+  return true;
+};
+
+export const validateEmploymentAndRespondentDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+
+  const { startDate, pastEmployer, isStillWorking, claimantWorkAddressQuestion, respondentEnterPostcode, noticeEnds } =
+    userCase;
+
+  if (pastEmployer === 'Yes') {
+    if (!startDate) {
+      return false;
+    }
+  }
+  if (isStillWorking === 'Notice') {
+    if (!startDate || !noticeEnds) {
+      return false;
+    }
+  }
+  return !(!pastEmployer || isStillWorking === undefined || !claimantWorkAddressQuestion || !respondentEnterPostcode);
+};
```

Comment:
Similar to the previous comment, we aren't checking what is mandatory. Mandatory questions in this section are the respondent details (name, addresss and ACAS cert)

<img width="697" alt="Screenshot 2025-04-10 at 09 13 50" src="https://github.com/user-attachments/assets/4fc3edee-5478-4237-ab88-3a767c279dac" />